DesignSafe-CI/portal
# Task/des 2849 cite this data

## Overview: ##
Add "Download Citation" with available download options: DataCite XML, RIS, BibTeX
Include the DownloadCount, ViewCount, CiationCount
Made DOI a link.

## PR Status: ##


## Related Jira tickets: ##

* [DES-2849](https://tacc-main.atlassian.net/browse/DES-2849)

## Summary of Changes: ##
I took the DownloadCount, ViewCount, CiationCount parts from the MetricsModal task I've been working on. I removed the actual modal and just left these 3 values. I also included the 3 citation download options. 

## Testing Steps: ##
1. Load a publication, verify the Downloads, Views, and Citations values load. ...OR they show '--' (no error occurs)
2. Verify the 3 citation download options work. 
3. Verify DOI link works

## UI Photos:
<img width="569" alt="Screen Shot 2024-05-30 at 1 58 35 PM" src="https://github.com/DesignSafe-CI/portal/assets/35277477/e14de638-4d28-4c62-b690-79fc8d2105f6">


## Notes: ##
In the code, I commented out some MetricsModal stuff, I need that for DES-2666
